beatrix/src/test/java/org/killbill/billing/beatrix/integration/TestTagApi.java

```diff
@@ -102,7 +102,7 @@ public void testApiTagOnInvoice() throws Exception {
         // Create a new tag definition
         //
         busHandler.pushExpectedEvents(NextEvent.TAG_DEFINITION);
-        final TagDefinition tagDefinition = tagUserApi.createTagDefinition("foo", "foo desc", ImmutableSet.<ObjectType>of(ObjectType.ACCOUNT, ObjectType.INVOICE), callContext);
+        final TagDefinition tagDefinition = tagUserApi.createTagDefinition("foo", "foo desc", new TreeSet<>(Set.of(ObjectType.ACCOUNT, ObjectType.INVOICE)), callContext);
```

Comment:
You might want to add a comment that the `TreeSet` is needed for the ordering.

Reply:
Done